types/src/v0/impls/header.rs

```diff
@@ -440,8 +440,11 @@ impl Header {
                 );
             } else {
                 ensure!(
-                    fee_account
-                        .validate_sequencing_fee_signature_marketplace(fee_signature, *fee_amount,),
+                    fee_account.validate_sequencing_fee_signature_marketplace(
+                        fee_signature,
+                        *fee_amount,
+                        height,
```

Comment:
I don't 100% know what this is but I thought we added the view number to a builder signature, not the height, so I want to double check this is correct